forge-gui/res/cardsfolder/upcoming/gray_harbor_merfolk.txt

```diff
@@ -0,0 +1,8 @@
+Name:Gray Harbor Merfolk
+ManaCost:1 U
+Types:Creature Merfolk Rogue
+PT:0/3
+K:Unblockable
+S:Mode$ Continuous | Affected$ Card.Self | AddPower$ 2 | IsPresent$ Creature.IsCommander+YouCtrl,Planeswalker.IsCommander+YouCtrl | Description$ CARDNAME gets +2/+0 as long as you control a commander that's a creature or planeswalker.
+DeckHints:Type$IsCommander
```

Comment:
Looks like this is a forgescribe artifact

The AI won't actually do anything with this, I don't think

Best bet again is `AI:RemoveDeck:NonCommander` I think

Reply:
I'll fix it in Tome of Legends